repose-aggregator/components/filters/rackspace-auth-user/src/test/scala/org/openrepose/filters/rackspaceauthuser/RackspaceAuthUserParsingTest.scala

```diff
@@ -148,11 +149,62 @@ class RackspaceAuthUserParsingTest extends FunSpec with Matchers {
             |</auth>
           """.stripMargin.trim()
 
-        val username = handler.username2_0XML(new ByteArrayInputStream(payload.getBytes))
+        val (domain, username) = handler.username2_0XML(new ByteArrayInputStream(payload.getBytes))
+        domain shouldBe None
         username shouldBe Some("nameOfTenant")
       }
-      //TODO: don't care about the multifactor parts...?
+
+      it("parses scope/username out of an MFA request") {
+        // The example this was taken from is currently incorrect.
+        // This is the best guess at what it should be until it is corrected.
+        // The error will only effect the outcome if/when we start differentiating Scope'd items.
+        // TODO: Have the API documentation updated and update this payload.
+        val payload =
+          """<?xml version="1.0" encoding="UTF-8"?>
+            |<auth xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+            |      xmlns="http://docs.openstack.org/identity/api/v2.0">
+            |    <RAX-AUTH:scope>SETUP-MFA</RAX-AUTH:scope>
+            |    <passwordCredentials username="demoAuthor" password="myPassword01"/>
```

Comment:
And the change in the response is triggered by the `RAX-AUTH:scope` being set to `SETUP-MFA`?